source/renderer/app/i18n/locales/en-US.json

```diff
@@ -250,7 +250,7 @@
   "settings.menu.display.link.label": "Themes",
   "settings.menu.general.link.label": "General",
   "settings.menu.support.link.label": "Support",
-  "settings.menu.termsOfUse.link.label": "Terms of use",
+  "settings.menu.termsOfUse.link.label": "Terms of service",
```

Comment:
@yakovkaravelov please update this key for Japanese translation too: https://github.com/input-output-hk/daedalus/blob/v2-integration/source/renderer/app/i18n/locales/ja-JP.json#L253